tests/main.tf

```diff
@@ -0,0 +1,37 @@
+module "mediaconvert_queue" {
+  source = "../."
+
+  queue_name = "test-queue"
+
+  rules = {
+    orders = {
+      name           = "test-mediaconvert"
+      description    = "Capture mediaconvert job events"
+      event_bus_name = "default"
+      event_pattern  = jsonencode({ "source" : ["aws.mediaconvert"] })
+
+      pattern = {
+        source        = ["aws.mediaconvert"]
+        "detail-type" = ["MediaConvert Job State Change"]
+        detail = {
+          userMetadata = {
+            env = ["DEV"]
+          }
+        }
+      }
+      is_enabled = true
+    }
+  }
+  targets = {
```

Comment:
target sqs can be generated inside module, this variable we can use to pass extra values,